### PR TITLE
Add documentation to prep for v2 release.

### PR DESCRIPTION
**Pull request by 0xekez**
This PR adds a README and documentation to every package and contract in the repository.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,11 +1,8 @@
 # Contributing to DAO DAO
 
-Thanks for your interest in contributing. We're excited to welcome you
-into our community. :)
-
-We value kindness, humility, and low-ego communication. You can read
-our [Code of Conduct](./CODE_OF_CONDUCT.md) for a poetic version of
-this.
+Thanks for your interest in contributing. We value kindness, humility,
+and low-ego communication. You can read our [Code of
```
Comment (blue-note) [inline CONTRIBUTING.md:4]: I like this :)

**Review by blue-note — APPROVED** [review]

File: contracts/external/cw-admin-factory/README.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # cw-admin-factory
 
-Serves as a factory that isntantiates contracts and sets them as their own wasm admins.
+Serves as a factory that isntantiates contracts and sets them as their
```
Comment (onewhiskeypls) [inline contracts/external/cw-admin-factory/README.md:3]: **nit**: instantiates :)

File: contracts/staking/cw20-stake-reward-distributor/README.md [hunk 3]
```diff
@@ -1,3 +1,4 @@
-# stake-cw20-reward-distributor
+# Stake CW20 Reward Distributor
```
Comment (onewhiskeypls) [inline contracts/staking/cw20-stake-reward-distributor/README.md:1]: **Question**: Should this be "CW20 Stake Reward Distributor" ?

File: contracts/voting/cwd-voting-cw20-staked/README.md [hunk 4]
```diff
@@ -1,40 +1,46 @@
...
-`TokenContract` - Provided via the `token_query` macro, simply returns the 
-underlying CW20 token's address.
+`TokenContract` - Provided via the `token_query` macro, simply returns
+the underlying CW20 token's address.
 
-`StakingContract` - Returns the underlying staking contract used to derive
-voting power at a given height. Should point to an instance of `stake-cw20`.
+`StakingContract` - Returns the underlying staking contract used to
+derive voting power at a given height. Should point to an instance of
+`stake-cw20`.
```
Comment (onewhiskeypls) [inline contracts/voting/cwd-voting-cw20-staked/README.md:25]: **question**: should this be "`cw20-stake`" ?

**Review by onewhiskeypls — COMMENTED** [review]
1 nit and 2 questions 👯 . 

@0xekez did you want a full reading of the readmes or just these changes?

**Review by NoahSaso — APPROVED** [review]

**Pull request merged**